web/tests/emoji_picker.test.cjs

```diff
@@ -47,9 +49,9 @@ run_test("initialize", () => {
     assert_emoji_category(complete_emoji_catalog.pop(), "fa-car", 195);
     assert_emoji_category(complete_emoji_catalog.pop(), "fa-hashtag", 224);
     assert_emoji_category(complete_emoji_catalog.pop(), "fa-smile-o", 169);
-    assert_emoji_category(complete_emoji_catalog.pop(), "fa-star-o", popular_emoji_count);
     assert_emoji_category(complete_emoji_catalog.pop(), "fa-thumbs-o-up", 386);
     assert_emoji_category(complete_emoji_catalog.pop(), "fa-lightbulb-o", 264);
+    assert_emoji_category(complete_emoji_catalog.pop(), "fa-star-o", popular_emoji_count);
```

Comment:
I feel like it'd be good to extend this test to set up some `emoji_frequency` data and be able to see the impact of that.

Also, why is this category now in a different place in the sorted list?

Reply:
Ahh, OK, great.